templates/ipset.service.epp

```diff
@@ -9,9 +8,6 @@ Documentation=https://github.com/voxpupuli/puppet-ipset
 <% if $firewall_service { -%>
 Before=<%= $firewall_service %>
 <% } -%>
-<% if $service_dependency { -%>
-Require=<%= $service_dependency %>
-<% } -%>
 After=network-online.target
```

Comment:
This is still correct?

Reply:
as mentioned in the commit text, I will update it in another PR. It works, but can be improved.